function-aws-api-proxy/src/main/java/io/micronaut/function/aws/proxy/MicronautLambdaContainerContext.java

```diff
@@ -40,9 +40,9 @@
     JsonMediaTypeCodec getJsonCodec();
 
     /**
-     * @return The Jackson's {@link ObjectMapper}
+     * @return The {@link JsonMapper}
      */
-    default ObjectMapper getObjectMapper() {
-        return getJsonCodec().getObjectMapper();
+    default JsonMapper getObjectMapper() {
```

Comment:
There is a `import io.micronaut.serde.ObjectMapper;` why not use instead of `JsonMapper`?

Reply:
JsonCodec returns a JsonMapper, which isn't an ObjectMapper 😔